src/Tribe/Attendee_Registration/Template.php

```diff
@@ -71,6 +71,16 @@ public function setup_context( $posts ) {
 			return $posts;
 		}
 
+		if ( wp_is_block_theme() ) {
+			error_log(
+				esc_html__(
+					'We detected that you are using a Full Site Editing theme. In order for the Attendee Registration Page to function properly, you will need to set up a page, using the [tribe_attendee_registration] shortcode in the attendee registration Attendee Registration settings.',
+					'event-tickets'
+				)
+			);
+			wp_die( 'A configuration error has occurred.' );
```

Comment:
Placeholder text, we may want to find better verbiage since this is a front facing issue that will occur until either the Theme is changed, or they make a custom page using our shortcode.